R/zzz.R

```diff
@@ -60,6 +60,9 @@ register_mlr3 = function() {
   # measures --------------------------------------------------------------------
   x = utils::getFromNamespace("mlr_measures", ns = "mlr3")
   x$add("forecast.mae", MeasureForecastMAE)
```

Comment:
So this is not really your fault, but if you want to, you can duplicate this PR from mlr3learners: https://github.com/mlr-org/mlr3learners/commit/195a4bb6c85f52c187d818a0e75ce94c10cb1683

I.e. the `.unOnload` should make sure that these dictonary items are also being removed when unloading the R package again.

Reply:
sounds good